Pipfile

```diff
@@ -4,36 +4,36 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-feedparser = "*"
-loggly-python-handler = "*"
-lxml = "*"
-psycopg2-binary = "*"
-maxminddb-geolite2 = "*"
-requests = "*"
-pycryptodome = "*"
+feedparser = "==6.0.10"
+loggly-python-handler = "==1.0.1"
+lxml = "==4.9.2"
+psycopg2-binary = "==2.9.5"
+maxminddb-geolite2 = "==2018.703"
+requests = "==2.28.1"
+pycryptodome = "==3.16.0"
 uszipcode = "==0.2.4"
-Flask = "*"
-Flask-Bcrypt = "*"
+Flask = "==2.1.3"
+Flask-Bcrypt = "==1.0.1"
 Flask-CORS = "==3.0.10"
-Flask-JWT-Extended = "==4.*"
-Flask-SQLAlchemy-Session = "*"
-Flask-Babel = "*"
-GeoAlchemy2 = "*"
-PyJWT = "*"
+Flask-JWT-Extended = "==4.4.4"
+Flask-SQLAlchemy-Session = "==1.1"
+Flask-Babel = "==2.0.0"
+GeoAlchemy2 = "==0.11.1"
+PyJWT = "==2.6.0"
 SQLAlchemy = "==1.3.23"
-Pillow = "*"
-pytest = "*"
-pytest-datadir = "*"
-bcrypt = "*"
-gunicorn = "*"
+Pillow = "==9.4.0"
+pytest = "==7.2.0"
+pytest-datadir = "==1.4.1"
+bcrypt = "==4.0.1"
```

Comment:
I'm worried about this one, it seems to _maybe_ cause some issues. Would there be an issue locking it to 3.x?